app/api/routers/breeze_buddy/leads/rbac.py

```
@@ -81,28 +81,28 @@ def validate_lead_read_access(
-        lead.merchant_id not in current_user.merchant_ids
-        and "*" not in current_user.merchant_ids
+        lead.reseller_id not in current_user.reseller_ids
+        and "*" not in current_user.reseller_ids
     ):
         logger.warning(
             f"User {current_user.username} attempted to {operation} lead {lead.id} "
-            f"for unauthorized merchant: {lead.merchant_id}"
+            f"for unauthorized reseller: {lead.reseller_id}"
         )
         raise HTTPException(
             status_code=status.HTTP_404_NOT_FOUND, detail=f"Lead not found"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)